roles/network_plugin/calico/tasks/install.yml

```diff
@@ -227,7 +227,7 @@
               "cidr": "{{ calico_pool_cidr | default(kube_pods_subnet) }}",
               "ipipMode": "{{ calico_ipip_mode }}",
               "vxlanMode": "{{ calico_vxlan_mode }}",
-              "natOutgoing": {{ nat_outgoing|default(false) and not peer_with_router|default(false) }}
```

Comment:
This original line has been added since https://github.com/kubernetes-sigs/kubespray/pull/3086
And I could not find the reason of this condition.
The new line
```
"natOutgoing": {{ nat_outgoing|default(false) }}
```
seems pretty straightforward for me.

Reply:
wow PR 3xxx 😄